package-lock.json

```diff
@@ -0,0 +1,6727 @@
+{
```

Comment:
package-lock shouldnt be commited (should be in gitignore, delete it - commit deleted, then add to giignore - and commit gitignore in this sequence)